exercises/word-search/example.js

```diff
@@ -0,0 +1,146 @@
+class WordSearch {
+  constructor(grid) {
+    this.grid = grid;
+  }
+
+  find(words) {
+    return words
+      .map(word => ({[word]: findWordInAnyDirection(word, this.grid)}))
+      .reduce((acc, oneWord) => Object.assign(acc, oneWord), {});
+  }
+}
+
+function findWordInAnyDirection(word, grid) {
```

Comment:
Wow! Function names make this piece of code really readable. That's a relief, because there's lots of code in this file (the exercise doesn't feel easy to implement).